content/articles/how-to-load-and-query-a-csv-file-created-with-python-in-ibm-db2-database/index.md

```diff
@@ -0,0 +1,176 @@
+
+### How to load and query a csv file created with python in IBM db2 database.
+
+#### In this article you will learn how to 
+- Create a CSV file in python
+- Load data into IBM Db2 database
+- Run SQL queries with your CSV dataset in IBM db2 database
+
+### Introduction
+A Comma Separated Values (CSV) file are semi-structure data files which can be transformed to structured data for an ordinary user to understand the meaning of the the data in it. This is where the IBM Db2 database comes in to play, it can collect semi-structure data and transform it into a structured data and also gives you the convenient to query the data to get the required result you need.
+
+### Prerequisite 
+- python 3.10 or newest version installed on your machine.
+- working knowledge on SQL and Python.
+- IBM Db2 database account (In this tutorial we will use the free version).
+
+### Creating a CSV file with python.
+They are various ways you can create a CSV file in python, but you can use this simple format provided in this article. Here is a simple way you can create your CSV file in python using the code as shown below.
+```python
+file = open("./demo.csv", "w")
+```
+If a `demo.csv` file does not exist in the file directory you are working with, this will generate one for you.
+Once your CSV file has been created you will need to put in some data in it, since it is empty. 
+
+For you to achieve that, you will need to change the file permission to read and writie(r+) so you can add some data into your CSV file as shown below.
+```python
+file = open("./data.csv", "r+")
+```
+
+Now you have changed the permission to read and write, you can add some data into your CSV file with python using the code as shown below:
+```python
+file.write("id,nameid,email\n")
+file.write("Ade,Adeda,email@adeda\n")
+file.write("King,Kingson,email@kingson\n")
+file.write("EFe,Mena,email@efe\n")
+file.write("Aisha,Zuberu,email@aisha\n")
+file.write("Ochuko,Onojakpor,email@ochuko\n")
+file.write("Doro,Onome,email@doro\n")
+file.write("Joel,Prince,email@orince")
+file.close()
+```
+
+At this point your CSV file should have your data in it which would be needed to query in your IBM Db2 database.
+Now your CSV file is ready, you should be able to view it in your Integrated Development Environment (IDE) as shown below.
+
+![democsvfile](democsv.jpg)
```

Comment:
The image path is incorrect. It must be something like `/engineering-education/[ARTICLE-DIRECTORY]/[IMAGE-NAME-WITH-EXTENSION]`

Reply:
Do this change for all the images.